streamlit-python/README.md

```diff
@@ -1,34 +1,33 @@
 ## Introduction
 
-This is an example template for a Snowflake Native Apps project which demonstrates the use of Python extension code and adding Streamlit code. This template is meant to guide developers towards a possible project structure on the basis on functionality, as well as indicate the contents of some common and useful files. 
+This is an example template for a Snowflake Native App project which demonstrates the use of Python extension code and adding Streamlit code. This template is meant to guide developers towards a possible project structure on the basis on functionality, as well as indicate the contents of some common and useful files. 
 
-Since this template contains python files only, you do not need to build the source code with any additional step. You can directly go to the next section. However, if there were any source code that needed to be built, you would manually perform the build steps here before proceeding to the next section. 
+Since this template contains python files only, you do not need to build the source code with any additional step. You can directly go to the next section. However, if there were any source code that needed to be built, you need to manually perform the build steps here before proceeding to the next section. 
```

Comment:
```suggestion
Since this template contains Python files only, you do not need perform any additional steps to build the source code. You can directly go to the next section. However, if there were any source code that needed to be built, you must manually perform the build steps here before proceeding to the next section. 
```